templates/blog/post_detail.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <h2>{{ post.title }}</h2>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+      integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+      crossorigin="anonymous">
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+  <p>{{ post.owner }} at {{ post.created_time }}</p>
+  <p>{{ post.content }}</p>
+  <p>{{ post.comm_post.count }} comment{{ post.comm_post.count|pluralize }}</p>
+  {% if post.comm_post.count > 0 %}
+    {%  for comment in post.comm_post.all %}
+      <table class="table">
+        <tr>
+          <td>{{ comment.user }} {{ comment.created_time }}</td>
+        </tr>
+      <tr>
+        <td>{{ comment.content }}</td>
+      </tr>
+    </table>
+    {% endfor %}
+  {% else %}
+    <p>There are no comments yet.</p>
+  {% endif %}
+  {% load crispy_forms_filters %}
+  {% if user.is_authenticated %}
+    <form action="" method="post" novalidate>
+      {% csrf_token %}
+        {{ form|crispy }}
+        <h6>Add a new comment</h6>
+        <label>
+          <input type="text" size="200">
```

Comment:
The input field for adding a new comment is missing a 'name' attribute. This attribute is necessary for the form data to be submitted correctly.